airflow/settings.py

```diff
@@ -71,6 +74,42 @@ def timing(cls, stat, dt):
 
 Stats = DummyStatsLogger
 
+
+def validate_stat(f):
```

Comment:
one suggestion: now we have `DummyStatsLogger` and `SafeStatsdLogger`, maybe we could move them to a separate file instead enlarging the setting.py file?

Reply:
Yup.